setup.py

```diff
@@ -28,8 +28,16 @@
     packages=['mdbenchmark'],
     package_data={'mdbenchmark': ['templates/*']},
     install_requires=[
-        'numpy>=1.8', 'mdsynthesis', 'click', 'jinja2', 'pandas', 'matplotlib',
-        'python-Levenshtein', 'xdg<2',
+        'numpy>=1.8',
+        'mdsynthesis',
+        'click',
+        'jinja2',
+        'pandas',
+        'matplotlib',
+        'python-Levenshtein',
+        'xdg<2',
     ],
     entry_points={'console_scripts': ['mdbenchmark=mdbenchmark.cli:cli']},
+    setup_requires=['pytest-runner'],
```

Comment:
setup doesn't need pytest

Reply:
I added that in case we use `python setup.py test` to run the tests, as I had some problems with setting up `pytest` initially.